### ISSUE TITLE
Questions About calculate_video_results in test.py

### ISSUE THREAD

**Issue by CLAPooo** [kenshohara/3D-ResNets-PyTorch#119]
In your paper, 
```
_We then input each clip into the
networks and estimate the clip class scores, which are averaged
over all the clips of the video._
```
but in your codes,  **just the clips of same label in a batch** are averaged, not all clipos of the video.
In addition, the new scores of the same label in a new batch will cover the score before.

> def calculate_video_results(output_buffer, video_id, test_results, class_names):
>     video_outputs = torch.stack(output_buffer)
>     average_scores = torch.mean(video_outputs, dim=0)
>     sorted_scores, locs = torch.topk(average_scores, k=10)
> 
>     video_results = []
>     for i in range(sorted_scores.size(0)):
>         video_results.append({
>             'label': class_names[locs[i]],
>             'score': sorted_scores[i]
>         })
> 
>     **test_results['results'][video_id] = video_results**

>         for j in range(outputs.size(0)):
>             if not (i == 0 and j == 0) and targets[j] != previous_video_id:
>                 calculate_video_results(output_buffer, previous_video_id,
>                                         test_results, class_names)
>                 output_buffer = []
>             output_buffer.append(outputs[j].data.cpu())
>             previous_video_id = targets[j]

Looking forward to your reply! Thank you!
@kenshohara
